core/dbt/cli/main.py

```diff
@@ -576,6 +577,36 @@ def run(ctx, **kwargs):
     return results, success
 
 
+# dbt run
+@cli.command("retry")
+@click.pass_context
+@p.project_dir
+@p.profiles_dir
+@p.vars
```

Comment:
I think we don't support `vars` in retry? As it might change the graph

Reply:
Or is the idea that if the user specified `vars` in the previous command, they have to specify it in the retry command also to make everything work?